docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,13 +3,27 @@ title: Vulnerability Scanning
 weight: 35
 ---
 
-Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. 
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). To be able to use Trivy, Clair or both you must have enabled Trivy, Clair or both when you installed your Harbor instance (by appending installation options `--with-trivy`, `--with-clair` or both).
 
 {{< important >}}
-Clair is an optional component. To be able to use Clair you must have enabled Clair when you installed your Harbor instance.
+So far, Harbor supports only one system default scanner. The following principles will be applied to determine the system default scanner among the default installed scanners.
+
+For a brand new installation case:
+
+If no scanner is installed, no system default scanner will be set then;
+
+If only one scanner (either Trivy or Clair) is installed, the installed one will become the system default scanner automatically;
+
+If both Trivy and Clair are installed, Trivy will be the system default scanner then.
+
+For an upgrading case:
+
+If the upgrading path is from the version that is >=V1.10 to current version (V2.0) and there was an existing system default scanner “ABC” is set in the previous version, no matter if it is Trivy or Clair, the scanner "ABC" will be kept as system default scanner as it was;
+
+Otherwise, Harbor will do the similar operation to the above brand new installation case.
 {{< /important >}}
 
-You can also connect Harbor to your own instance of Clair or to additional vulnerability scanners by using an interrogation service. You configure additional scanners in the Harbor interface, after you have installed Harbor. For the list of additional scanners that are currently supported, see the [Harbor Compatibility List](../../install-config/harbor-compatibility-list.md#scanner-adapters).
+You can also connect Harbor to your own instance of Trivy/Clair or to other additional vulnerability scanners by using an interrogation service. You configure additional scanners in the Harbor interface, after you have installed Harbor. For the list of additional scanners that are currently supported, see the [Harbor Compatibility List](../../install-config/harbor-compatibility-list.md#scanner-adapters).
```

Comment:
'by using an..' -> 'through Harbor's embedded interrogation service'

Reply:
'You configure additional scanners in the Harbor interface, after you have installed Harbor. ' -> ' These scanners can be configured in the Harbor UI at any time after installation'